.travis.yml

```diff
@@ -80,6 +93,15 @@ install:
   - travis_retry composer update --no-interaction $DEPENDENCIES
 
 before_script:
+  - if [ "$RUN_PHPUNIT_PATCHES" = "true" ]; then
```

Comment:
What about moving this to separate shell script to make the config shorter?

Also we may eg. soon migrate to GH Actions only, so there won't be a need to copy-paste it.

Reply:
@williamdes can you maybe check in on this one?

Reply:
Done in 720c2750d4eb3064e6363f3f7a7c16fba75cca59

Reply:
Just a really minor thing but maybe name the directory `bin/` so it's more in line with other composer libraries?

Reply:
I would say no, on [Doctum](https://github.com/code-lts/doctum/tree/main/bin) and some other libs at [phpMyAdmin](https://github.com/phpmyadmin/sql-parser/tree/master/bin) scripts found in bin are for users of the lib :)

Reply:
Gotcha 👍